### PR TITLE
update issue template with some background

### PR DESCRIPTION
**Pull request by jospoortvliet**
add section with some good reads for people reporting bugs...

Thoughts? Should we put this in other repo's too?

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 1]
```diff
@@ -9,6 +9,12 @@ Thanks for reporting issues back to Nextcloud!
...
 
+Note: Try to be friendly and patient, we will try, too. Nextcloud is a volunteer community. While some people are paid to work on Nextcloud, they are paid by customers, so unless you are a customer, nobody gets paid to help you. People help you because they are nice, so treat them as such.
+
+We get a LOT of bug reports and we have to focus on the most important ones, that is, those with the most severe impact, no work-around and many users who encounter it. If we don't fix a problem that bothers you, it is not that we are stupid or that we hate you, it is [because we have many, many things on our plate.](https://erikbern.com/2020/03/10/never-attribute-to-stupidity-that-which-is-adequately-explained-by-opportunity-cost.html)
```
Comment (nickvergessen) [inline .github/ISSUE_TEMPLATE/Bug_report.md]: markdown doesnt work in the input :sweat_smile:

**Comment by kesselb** [comment]
Mind to review https://github.com/nextcloud/server/pull/19587 first? The bigger issue from my pov is that people are don't know how to search properly and are not familiar with GitHub. 

While those articles are nice to read I guess one who wants to report a issue does not want to read 3 other pages first. To be honest I find some of those changes are very unwelcoming. The sentence about paid support is important and I like that clarification about priorities.

**Comment by nickvergessen** [comment]
@jospoortvliet can you rebase?

**Comment by ChristophWurst** [comment]
Closing due to inactivity. @jospoortvliet please rebase and re-open if you would like to still get this in :v:

**Pull request closed without merging**